.env

```diff
@@ -0,0 +1,4 @@
+# Update these with your Supabase details from your project settings > API
+# https://app.supabase.com/project/_/settings/api
+NEXT_PUBLIC_SUPABASE_URL=https://qvqhxbmnksmbbubvgqhc.supabase.co
```

Comment:
already have supabase project deployed and connected. share email then i can add as dev